lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -64,6 +64,7 @@ def authorize(money, payment, options = {})
           r.process do
             post = create_post_for_auth_or_purchase(money, payment, options)
             post[:capture] = "false" unless emv_payment?(payment)
```

Comment:
The unless/if right next to each other points to this maybe being better as an if/else